src/api/routes/user.ts

```diff
@@ -466,4 +466,34 @@ export default (app: Router): void => {
         authenticateToken,
         userController.readBeneficiaryRules
     );
+
+    /**
+     * @swagger
+     *
+     * /user/read-rules/manager:
+     *   put:
+     *     tags:
+     *       - "user"
+     *     summary: Read manager rules
+     *     responses:
+     *       "200":
+     *         description: OK
+     *         content:
+     *           application/json:
+     *             schema:
+     *               type: object
+     *               properties:
+     *                 success:
+     *                   type: boolean
+     *                 data:
+     *                   type: boolean
+     *     security:
+     *     - api_auth:
+     *       - "write:modify":
+     */
+     route.put(
+        '/read-rules/manager',
+        authenticateToken,
+        userController.readManagerRules
+    );
```

Comment:
What if we do `/read-rules/(beneficiary|manager)` and then merge into one endpoint and one controller method which chooses the service?
reference https://expressjs.com/en/guide/routing.html

Reply:
Much better!
I changed. Is this the way you thought?

Reply:
Yes, it is :relaxed: looks nice